src/selectedtests/task_mappings/create_task_mappings.py

```diff
@@ -47,8 +48,12 @@ def generate_task_mappings(
     :return: An instance of TestMappingsResult and the most recent version analyzed during analysis.
     """
     source_re = re.compile(source_file_pattern)
-    module_source_re = None
+    module_source_re = re.compile("")
```

Comment:
I think match gets called on this downstream no matter what --> or I've missed something.